src/main/java/tempest/ui/pages/HomePage.java

```diff
@@ -13,19 +12,21 @@
 
     private final GUIManager manager;
 
-    private JButton manageModulesButton;
-    private JButton addSessionButton;
+    private LinkButton manageModulesLink = new LinkButton("Add a new module", PageNames.MANAGE_MODULES, this);
+    private LinkButton addSessionLink = new LinkButton("Add a new session", PageNames.ADD_SESSION, this);
+    private LinkButton chartsLink = new LinkButton("View Data", PageNames.CHART_VIEW, this);
 
     public HomePage(GUIManager guiManager) {
         super();
         this.manager = guiManager;
+        addNavButtons();
+    }
 
+    private void addNavButtons() {
         JPanel buttonPanel = new JPanel();
-        manageModulesButton = new LinkButton("Add a new module", PageNames.MANAGE_MODULES, this);
-        addSessionButton = new LinkButton("Add a new session", PageNames.ADD_SESSION, this);
```

Comment:
Any particular reason for having the LinkButtons initialised at the top? I'm assuming just your preference

Reply:
I guess it is a bit neater in the method